<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDownMenuWidget</class>
 <widget class="QWidget" name="BaseDownMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="BaudLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>波 特 率:</string>
   </property>
  </widget>
  <widget class="QLabel" name="surveyModeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>采集模式:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cycleTimeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>148</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>采集频率:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
  <widget class="QLabel" name="cycleTimeUnitLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>148</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="saveModeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>存储模式:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
